flink-core/src/test/java/org/apache/flink/api/common/typeutils/ComparatorTestBase.java

```diff
@@ -45,6 +45,10 @@
 	// Same as in the NormalizedKeySorter
 	private static final int DEFAULT_MAX_NORMALIZED_KEY_LEN = 8;
 
+	protected boolean[] getTestedOrder() {
+		return new boolean[] {true, false};
```

Comment:
Recently someone told me that `booleans` are hard to maintain because you don't immediately see what the `true` /`false` really means. You could use `Order.ASCENDING` and `Order.DESCENDING` here instead.

Reply:
(it was @tillrohrmann, btw)